services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -918,8 +918,18 @@ module.exports = {
         excludeContentTypes,
       });
 
-      customAttributes.forEach(({ key, value }) => {
-        query.$and.push({ [`customAttributes.${key}`]: value });
+      customAttributes.forEach(({ key, value, exists }) => {
+        const valueOrExists = Boolean(value || typeof exists !== 'undefined');
+        if (!valueOrExists) {
+          throw new UserInputError('Either value or exists must be defined');
+        } else {
+          query.$and.push({
+            [`customAttributes.${key}`]: {
+              ...((value && exists !== false) ? { $nin: ['', null], $exists: true } : { $in: ['', null] }),
+            },
+          });
+          query.$and.push({ [`customAttributes.${key}`]: value });
```

Comment:
Since we're changing it up with the exists operators, use `$eq` for value to keep it consistent. Something like this:
```js
[cattr.${key}]: {
  ...(value && { $eq: value }),
  ...(exists === false && { $exists: false, $nin, $in, ... }),
  ...(exists === true && { $exists: true, ...  }),
```